contents/en/rules_en.md

```diff
@@ -1,52 +1,52 @@
+|The violation always will be considered in the context of earlier events. Usually, the one who generates the situation is the one who takes responsibility.|
+|Full isolation of a player is equivalent to killing them.|
+|Infliction of grievous bodily harm, which causes long healing in medbay (more than 30 minutes) is punished half as severely as murder.|
 
 |5. Do not use IC knowledge, which is unknown to your character.|
 | :------------ |
-|Do not use IC knowledge that you knew by ООС-way.|
-|Do not use other roles' difficult devices. Base operations, like wall dismantling, are free for all, but surgery and engine fitting could be done only by specialists. With the exception of related posts, like robotics, or Jacks-of-all-trades: scientists and RD.|
-|The crew is aware of existing of all special roles and have approximate information about their methods, but doesn't know anything about their skills and devices.|
+|Do not use knowledge gained from OOC sources in-game.|
+|Do not use other roles' difficult devices. Basic operations, like wall dismantling, are known by all jobs, but surgery and the engine should be limited to specialists. With the exception of related jobs, like robotics, or Jacks-of-all-trades: scientists and the RD.|
+|The crew is aware of the existence of all special roles and have approximate information about their methods, but doesn't know anything about their skills and devices.|
 
 |6. Breaking the immersion.|
 | :------------ |
-|Your manner of speaking must fit to your roleplaying, but not to your regular chat-behavior. Ideally, your speech should be literary as much as possible. Without any emojis, internet-slang or text-cuts: your characters are communicate by voice.|
-|Do not interrupt the setting and lore.
-|When you use flavor text, you don't allow to write character, story or something like this. Only your appearance (and some OOC notes if it's needed).
-|Do not take the famous people names (Adolf Hitler) and too much popular characters (Darth Wader). You can roleplaying not so famous heroes, but you should agree it with administration first.|
+|Your manner of speaking must fit to how you are roleplaying, but not to how you normally behave in internet chatrooms and the like. Ideally, your speech should try to fit within normal grammar rules and literary standards as much as possible. Without any emojis, internet-slang or text-cuts: your characters communicate via voice.|
+|Do not act in a way that disrupts the setting and lore.
+|When you use flavor text, do not use it to write information about your character, their story or anything of that sort. Only your appearance (and some OOC notes if it's needed).
+|Do not take names of famous figures (Adolf Hitler) or popular characters (Darth Wader). You can roleplay as less famous heroes, but you should discuss it with the administration first.|
```

Comment:
Might be a sophistication on my part, but using "roleplay as less famous heros" implies that villains, anti-heroes, and etc-etc can't be roleplayed as. Use "figures"/ "characters" instead?